heroku deploy error

```
2020-04-07T10:47:02.569724+00:00 app[web.1]: npm ERR! code ELIFECYCLE
2020-04-07T10:47:02.570227+00:00 app[web.1]: npm ERR! errno 1
2020-04-07T10:47:02.571994+00:00 app[web.1]: npm ERR! express-gen-ts@0.0.0 start: `node -r module-alias/register ./dist`
2020-04-07T10:47:02.572277+00:00 app[web.1]: npm ERR! Exit status 1
2020-04-07T10:47:02.572582+00:00 app[web.1]: npm ERR! 
2020-04-07T10:47:02.572740+00:00 app[web.1]: npm ERR! Failed at the express-gen-ts@0.0.0 start script.
2020-04-07T10:47:02.572936+00:00 app[web.1]: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-04-07T10:47:02.582859+00:00 app[web.1]: 
2020-04-07T10:47:02.583233+00:00 app[web.1]: npm ERR! A complete log of this run can be found in:
2020-04-07T10:47:02.583502+00:00 app[web.1]: npm ERR!     /app/.npm/_logs/2020-04-07T10_47_02_574Z-debug.log
```

--- a/tsconfig.prod.json
+++ b/tsconfig.prod.json
@@ -3,5 +3,5 @@
     "compilerOptions": {
         "sourceMap": false
     },
-    "exclude": ["spec", "app/**/*.mock.ts", "app/public/"]
+    "exclude": ["tests", "app/public/"]
 }

